cfn-resources/util/util.go

```diff
@@ -47,9 +48,29 @@ func CreateMongoDBClient(publicKey, privateKey string) (*mongodbatlas.Client, er
 	// Initialize the MongoDB Atlas API Client.
 	atlas := mongodbatlas.NewClient(client)
 	atlas.UserAgent = "mongodbatlas-cloudformation-resources/" + Version
+
+	basePath, err := newBasePath()
+
+	if err != nil {
+		return nil, err
+	}
+
+	if basePath != nil {
+		atlas.BaseURL = basePath
+	}
+
 	return atlas, nil
 }
 
+func newBasePath() (*url.URL, error) {
+	atlasURL := os.Getenv("MCLI_OPS_MANAGER_URL")
```

Comment:
let's go for the newer prefix 
```suggestion
	atlasURL := os.Getenv("MONGODB_ATLAS_OPS_MANAGER_URL")
```

(ugh I have so many regrets on using OPS_MANAGER_URL)

Reply:
Sure 👍 . We still use [MCLI_OPS_MANAGER_URL](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/a848e8e0a07eab64704f180583aa2194aaab7d83/mongodbatlas/provider.go#L77) in the terraform provider. I will open a follow-up PR on the provider to update the env var name there if that is okay with you.

Reply:
oh nvm. I see that supports both

Reply:
Updated